### ISSUE TITLE
[compatibility] with Inventorio

### ISSUE THREAD

**Issue by UplandJacob** [LegoMaster3650/Inventory-Crafter#7]
2 things: 
 * When a crafting table is in your Inventorio inventory, you can not use the 'inventory craft' button (on the vanilla inventory screen and of course with 'require crafting table' enabled)
 * since Inventorio uses a different inventory screen, the 'inventory crafting' button doesn't show up (can still access the vanilla inventory screen, but that is very inconveniant), I already have fixed his with FancyMenu by using the 'mimic vanilla button' feature, but it bugs out without a crafting table in your inventory.

In short, it would be great if you could have a crafting table in your extra inventory rows and still be able to use the inventory crafting and have the 'inventory crafting' button available in the Inventorio screen.

**Comment by LegoMaster3650** [comment]
I like inventorio so I'll consider this (but if I do I'll have to specifically add and maintain compatibility with this one mod)

**Comment by UplandJacob** [comment]
May have found what you need to implement this:

https://github.com/RubixDev/Inventorio/blob/main/src/main/kotlin/de/rubixdev/inventorio/player/inventory/PlayerInventoryExtension.kt

Line 191 has “contains(ItemTag)” like what you have used for the vanilla inventory

That code is incredibly confusing, so hopefully I found something useful. :)

**Comment by LegoMaster3650** [comment]
Don't worry I'll figure it out if I do it, thanks though! (might take a while to get to this though)

**Comment by LegoMaster3650** [comment]
Although yeah that's in kotlin which I've never tried to add compatibility with a kotlin mod so hopefully it won't be too hard.

**Comment by UplandJacob** [comment]
Yea I noticed that too.


For checking if the current screen is the inventorio screen:

"instanceof InventorioScreen" etc

class is declared here:
https://github.com/RubixDev/Inventorio/blob/main/src/main/kotlin/me/lizardofoz/inventorio/client/ui/InventorioScreen.kt

**Comment by LegoMaster3650** [comment]
Ok enough I can read the code and the hard part is making sure I don't break the mod for people without inventorio, reading code is the easy part.
